sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -175,18 +169,16 @@ def recovery_level(self):
     @property
     def tags(self):
         # type: () -> Dict[str, str]
-        """
-        Application specific metadata in the form of key-value pairs
+        """Application specific metadata in the form of key-value pairs
 
-        :rtype: dict
+        :rtype: dict[str, str]
         """
         return self._tags
 
     @property
     def managed(self):
         # type: () -> bool
-        """
-        Returns if the key's lifetime is managed by key vault
+        """Returns if the key's lifetime is managed by key vault
```

Comment:
English nitpick: returning is unconditional, so we should write 'whether' instead of 'if'.

Reply:
makes sense, i've changed it